content/10-managing-databases/07-introduction-database-caching.mdx

```diff
@@ -0,0 +1,97 @@
+
+A database caching strategy assists your primary database by easing the burden it might carry. This is most commonly seen by the rerouting of queries for frequently read data to the cache itself, rather than the primary database. The cache itself, resides in either the database, application, or even as a standalone access layer. 
+
+For example, your application requests user information from the database for the first time, this request goes from server to server and returns back the requested information. With caching, this user profile is stored closer to the requester after initial read, and there is a significant reduction in query processing time and database workload for all subsequent read requests for that data.
+
+## What are the benefits of database caching?
+
+Data retrieval speed greatly affects the user experience of an application. Implementing a caching strategy on your database can result in improved database performance, availability, and scalability for minimal cost depending on the strategy, all factors contributing to an overall positive application experience.
+
+### Performance
+As touched on briefly, database caching improves the performance of a database by making data more easily accessed. The cache acts has a sort of “keyboard short-cut” or “hot-key” for the application to reference data that it frequently is calling upon. 
+
+This speedier request can minimize the workload of the database, keeping it from spending inefficient amounts of time doing repetitive tasks. Instead making these tasks more efficient and simplifying data access. 
+
+### Availability
+While not a 100% failover strategy, caching also provides benefits to overall database availability. Depending on where the cache is stored, the cache can still provide a place for the application to call upon for data in the case of the primary database server becoming unavailable for any reason.
+
+While database performance is generally the primary reason for adopting a caching strategy, you also have the added benefit of some additional resiliency in the case of any backend failures. 
+
+### Scalability
+Similarly to added high availability, database caching has a positive effect on scalability. While it shouldn’t be your main consideration for a database scaling strategy, implementing caching to improve database performance reduces your database workload, therefore distributing backend queries across entities. 
+
+This distribution lightens the load on a primary database and can reduce costs and provide more flexibility in the processing of your data. This results in a more scalable environment as you continue for take more frequently requested data off the plate of the primary database.
+
```

Comment:
"...as you continue for take more..." sounds like it could use another look.

Also, somewhat unrelatedly, I think the affect that caching can have on scale is a really good one to make, but it may be a bit easier if you frame it as alleviating or delaying the need to scale.  Kind of a "do more with the resources you already have" strategy to help you put off further scaling which might be more costly.  Just something I was thinking about as I read through this.